presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -210,8 +230,12 @@
     private PrestoS3AclType s3AclType;
     private boolean skipGlacierObjects;
     private boolean requesterPaysEnabled;
+    private boolean streamingUploadEnabled;
+    private long streamingUploadPartSize;
     private PrestoS3StorageClass s3StorageClass;
 
+    private final ListeningExecutorService uploadExecutor = listeningDecorator(newCachedThreadPool(threadsNamed("s3-upload-%s")));
```

Comment:
I think we can make this just `ExecutorService` (and remove `listeningDecorator`) since we only call `get()` on the future and don't use the listening part.